src/_tutorials/dart-vm/httpserver.md

```diff
@@ -930,40 +932,48 @@ calls `bindSecure()` using
 a certificate created by the Dart team for testing.
 You **must** provide your own certificates for your servers.
 
+<?code-excerpt "httpserver/bin/hello_world_server_secure.dart" replace="/\S.*\/\*\d\*\//[!$&!]/g"?>
 {% prettify dart %}
-   import 'dart:io';
-
-   main() async {
-     var certificateChain =
-         Platform.script.resolve('server_chain.pem').toFilePath();
-     var serverKey =
-         Platform.script.resolve('server_key.pem').toFilePath();
-[[note]]1[[/note]]  [[highlight]]var serverContext = new SecurityContext();[[/highlight]]
-[[note]]2[[/note]]  [[highlight]]serverContext.useCertificateChain(certificateChain);[[/highlight]]
-[[note]]3[[/note]]  [[highlight]]serverContext.usePrivateKey(serverKey, password: 'dartdart');[[/highlight]]
-
-[[note]]4[[/note]]  var requests = await HttpServer.bindSecure('localhost', 4047, [[highlight]]serverContext[[/highlight]]);
-     print('listening');
-     await for (HttpRequest request in requests) {
-       request.response..write('Hello, world!')
-                       ..close();
-     }
-   }
+import 'dart:async';
+import 'dart:io';
+
+Future main() async {
+  var certificateChain =
+      Platform.script.resolve('server_chain.pem').toFilePath();
+  var serverKey = Platform.script.resolve('server_key.pem').toFilePath();
+  [!var serverContext = new SecurityContext(); /*1*/!]
```

Comment:
This new syntax is so much easier for me to read!

Reply:
I'm very glad that you like it. I juggled with a few options, but this was the simplest to deal with and using /*#*/ seems natural enough a marker for code.